src/core/1/guide/guides/essentials/plugins/index.md

```diff
@@ -66,22 +63,20 @@ Go to the Kuzzle installation folder and type:
 
 ```bash
 # Open plugins/available folder
-cd plugins/available
+cd <kuzzle directory>/plugins/available
 
 # Download Plugin to plugins/available folder
-git clone https://github.com/kuzzleio/kuzzle-core-plugin-boilerplate.git
+git clone git@github.com:kuzzleio/kuzzle-core-plugin-boilerplate
```

Comment:
```suggestion
git clone https://github.com/kuzzleio/kuzzle-core-plugin-boilerplate.git
```
IMHO it's better to keep the https link since it's only to download the repository the first time.  
SSH may be blocked by enterprise firewall or free internet provider

Reply:
I didn't know of that :+1: 
I'll revert to the HTTP link for all URLs I changed.